mdr/mdr.py

```diff
@@ -110,7 +143,11 @@ def score(self, features, classes):
             The estimated accuracy based on the constructed feature
 
         """
-        return 0.0
+        new_feature = self.fit_transform(features, classes)
+        results = (new_feature == classes).all()
```

Comment:
I believe `.all()` reduces the vector down to a single value. We don't want that here, as we're summing the vector in the next line.